### PR TITLE
OP-11496 : Added Maping URL for pagination APIs in dashboard, platform and SAPOR

### PR DESCRIPTION
**Pull request by sriharshakancharla**
https://devopsmx.atlassian.net/browse/OP-11496

We prefer small, well tested pull requests.

Please refer to [Contributing to Spinnaker](https://spinnaker.io/community/contributing/).

When filling out a pull request, please consider the following:

* Follow the commit message conventions [found here](https://spinnaker.io/community/contributing/submitting/).
* Provide a descriptive summary for your changes.
* If it fixes a bug or resolves a feature request, be sure to link to that issue.
* Add inline code comments to changes that might not be obvious.
* Squash your commits as you keep adding changes.
* Add a comment to @spinnaker/reviewers for review if your issue has been outstanding for more than 3 days.

Note that we are unlikely to accept pull requests that add features without prior discussion. The best way to propose a feature is to open an issue first and discuss your ideas there before implementing them.

### CODE REVIEW

**Comment by gopaljayanthi** [comment]
This PR  triggered this build in this execution, please merge to master only if this succeeds:  https://spin.prod.opsmx.com/#/applications/gcp-gitprtrigger/executions/details/01FTE8SNFF6K0CFN768DRQ1N4K

**Comment by gopaljayanthi** [comment]
This PR  triggered this build in this execution, please merge to master only if this succeeds:  https://spin.prod.opsmx.com/#/applications/gcp-gitprtrigger/executions/details/01FTFSGQBJ4MX30JNZWY8PXBKB

File: gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/internal/OpsmxDashboardService.groovy [hunk 1]
```diff
@@ -23,7 +23,11 @@ interface OpsmxDashboardService {
 
   @GET("/dashboardservice/{version}/{type}")
   Object getDashboardResponse1(@Path('version') String version,
-                              @Path('type') String type)
+                               @Path('type') String type,
+                               @Query("datasourceType") String datasourceType,
+                               @Query("pageNo") String pageNo,
+                               @Query("pageLimit") String pageLimit,
```
Comment (Pranav-b-7) [inline gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/internal/OpsmxDashboardService.groovy:29]: pageNo and pageLimit has to be Integer rit. ?

**Review by Pranav-b-7 — CHANGES_REQUESTED** [review]

**Comment by gopaljayanthi** [comment]
This PR  triggered this build in this execution, please merge to master only if this succeeds:  https://spin.prod.opsmx.com/#/applications/gcp-gitprtrigger/executions/details/01FTFXM2T2PQ2CZRNPEN66Z4WM

File: gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/internal/OpsmxDashboardService.groovy (hunk 1 above)
Comment (sriharshakancharla) [reply gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/internal/OpsmxDashboardService.groovy]: Yes. Changed to Integer

**Review by Pranav-b-7 — APPROVED** [review]

**Pull request merged**